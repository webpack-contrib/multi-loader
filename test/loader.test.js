/* eslint-disable
  import/order,
  comma-dangle,
*/
import webpack from './helpers/compiler';

import multi from '../src';

describe('Loader', () => {
  test('Default', async () => {
    const config = {
      rule: {
        test: /\.css$/,
        use: multi('css-loader', 'raw-loader'),
      }
    };

    const stats = await webpack('index.js', config);

    const src = stats.toJson()
      .modules[1].source
      .replace(new RegExp(`${process.cwd()}`, 'g'), '.');

    return expect(src).toMatchSnapshot();
  });
});
