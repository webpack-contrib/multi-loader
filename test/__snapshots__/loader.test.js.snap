// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Default 1`] = `
"require(\\"-!css-loader!./test/fixtures/index.css\\");
module.exports = require(\\"-!raw-loader!./test/fixtures/index.css\\");"
`;
